src/dev/bazel/pkg_npm_types/package_json.mustache

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "{{{NAME}}}",
```

Comment:
```
~/dev/kibana/node_modules/@types/elastic__datemath(pr/116465*) » cat package.json                                                                                                                          jon@xps
{
  "name": "@types/elastic__datemath",
  "description": "Generated by Bazel",
  "types": "./index.d.ts",
  "private": true,
  "license": "MIT",
  "version": "1.1.0"
}
```

Should license/version match the package's settings?

Reply:
This is a placeholder and those packages won't be published, I believe we can ignore those 😃